api/krusty/remoteload_test.go

```diff
@@ -94,15 +104,14 @@ func isLocalEnv(require *require.Assertions) bool {
 func runResourceTests(t *testing.T, cases map[string]*remoteResourceCase) {
 	t.Helper()
 
-	req := require.New(t)
 	for name, test := range cases {
 		savedTest := test // test assignment changes; need assignment in this scope (iteration) of range
 		t.Run(name, func(t *testing.T) {
-			if savedTest.local && !isLocalEnv(req) {
+			if savedTest.local && !isLocalEnv(t) {
 				t.SkipNow()
 			}
 			configureGitSSHCommand(t)
-			testRemoteResource(req, test)
+			testRemoteResource(t, test)
```

Comment:
Thank you so much for fixing the sub-test fails parent test bug! I don't think this caused the flakes, but can we change `test` to `savedTest` here? This was also a bug I introduced.

Reply:
Ah I see. These tests don't run in parallel so it makes no difference. I removed the assignment to `savedTest` in c6520a105